ref/general/mp-openapi.adoc

```diff
@@ -75,27 +75,36 @@ For example, the `@Operation` annotation provides more meaningful information ab
               $ref: '#/components/schemas/InventoryList'
 ----
 
-The information provided through the OpenAPI annotations augment the basic API documentation that is generated by the JAX-RS framework.
+The information that is provided through the OpenAPI annotations augments the basic API documentation that is generated by the JAX-RS framework.
 
-See the link:https://www.openliberty.io/docs/ref/microprofile/3.0/#package=org/eclipse/microprofile/openapi/annotations/package-frame.html&class=org/eclipse/microprofile/openapi/annotations/Operation.html[MicroProfile OpenAPI Javadoc] for the annotations available.
+See here for the link:https://github.com/eclipse/microprofile-open-api/wiki/Annotation-Samples[annotations available].
 
 You can further use link:http://download.eclipse.org/microprofile/microprofile-open-api-1.1/microprofile-openapi-spec.html#_filter[filters] to organize your APIs, for example, choose to expose some APIs to one audience and other APIs to another audience.
 
 == Design-first approach
 
-An alternative approach is to design the REST API in an editor, such as the Swagger editor, before writing any code.
-This approach enables you to spot any problems in the design before, rather than after, implementation.
-In large companies, it is important to have consistent, usable APIs that have been reviewed and agreed by the relevant people that it is a good API design.
+An alternative approach is to design the REST API in an editor, such as the Swagger editor, before any code.
+With this approach, you can spot any problems in the design before implementation.
+In large companies, it is important to have consistent, usable APIs reviewed and agreed by the relevant people that it is a good API design.
 This API design then forms a contract and must be implemented as agreed.
 
-You can write this API design in YAML or JSON which can then be used to generate the documentation and, optionally, even stubs for the API code itself.
-You can then also augment the documentation with annotations in the code as described previously.
+You can write this API design in YAML or JSON, which can then be used to generate the documentation and, optionally, even stubs for the API code itself.
+You can then also augment the documentation with annotations in the code as described.
 
 For examples of Static OpenAPI files, see link:https://github.com/eclipse/microprofile-open-api/wiki/Static-File-Samples[Static File Samples].
```

Comment:
When introducing links, we typically use the syntax "For more information, see" to maintain consistency.